components/sections/Footer.tsx

```diff
@@ -42,6 +42,10 @@ const footerContext = {
       url: 'https://opensauced.pizza/learn',
       label: 'opensauced.pizza/learn',
     },
+    {
+      url: 'https://opensauced.pizza/cli',
+      label: 'opensauced.pizza/cli',
+    },
```

Comment:
```suggestion
```
per @BekahHW's comment this is not necessary in the footer. More sauce is all external links.